.github/workflows/ci.yml

```diff
@@ -208,6 +208,19 @@ jobs:
             source plugin/trino-hive-hadoop2/conf/hive-tests-${{ matrix.config }}.sh &&
               plugin/trino-hive-hadoop2/bin/run_hive_s3_tests.sh
           fi
+      - name: Run Hive S3 Select JSON Tests
+        if: matrix.config == 'config-hdp3'
+        # JsonSerde class needed for the S3 Select JSON tests is only available on hdp3.
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESSKEY }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRETKEY }}
+          S3_BUCKET: "presto-ci-test"
+          S3_BUCKET_ENDPOINT: "s3.us-east-2.amazonaws.com"
+        run: |
+          if [ "${AWS_ACCESS_KEY_ID}" != "" ]; then
+            source plugin/trino-hive-hadoop2/conf/hive-tests-${{ matrix.config }}.sh &&
+              plugin/trino-hive-hadoop2/bin/run_hive_s3_select_json_tests.sh
```

Comment:
This is unnecessary complexity for ci.yml and quite an overhead if all we want is to run a test method under some configs only.

This perhaps could be done more simply 

- define a profile in pom.xml which would disable the JSON test
- change `hive-tests-${{ matrix.config }}.sh` so they populate some config-related profile

would that work?

Reply:
Hi, not sure I follow - so in this PR, I've defined a new profile in pom.xml to run the JSON test:

```
<profile>
            <id>test-hive-hadoop2-s3-select-json</id>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-surefire-plugin</artifactId>
                        <configuration>
                            <includes>
                                <include>**/TestHiveFileSystemS3SelectJsonPushdown.java</include>
                            </includes>
                        </configuration>
                    </plugin>
                </plugins>
            </build>
        </profile>
```
This test is now excluded from the Hive S3 tests profile.

In ci.yml, I initially had the following:

```
source plugin/trino-hive-hadoop2/conf/hive-tests-config-hdp3.sh &&
              plugin/trino-hive-hadoop2/bin/run_hive_s3_select_json_tests.sh
```

But switched it to the following based on Jan's feedback to be consistent with the other tests.

```
if: matrix.config == 'config-hdp3'
source plugin/trino-hive-hadoop2/conf/hive-tests-${{ matrix.config }}.sh &&
              plugin/trino-hive-hadoop2/bin/run_hive_s3_select_json_tests.sh
```

Are you referring to switching back to the first approach or something else?

Reply:
> source plugin/trino-hive-hadoop2/conf/hive-tests-${{ matrix.config }}.sh &&
>               plugin/trino-hive-hadoop2/bin/run_hive_s3_select_json_tests.sh

I am not convinced we need a separate run of tests, do we?

Reply:
Yes, that would make sense to do.